Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Exporters/TextSolutionFileParser.cs

```diff
@@ -0,0 +1,395 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
nit: There's a newer copyright header

Reply:
Added.

---
In reply to: [364492202](https://github.com/microsoft/MSBuildForUnity/pull/96#discussion_r364492202) [](ancestors = 364492202)